sci-biology/aghermann/aghermann-1.0.3.ebuild

```diff
@@ -12,17 +12,16 @@ LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-RDEPEND="
-	dev-lang/lua:0
-	dev-libs/libconfig
-	dev-libs/libconfig
-	dev-libs/libunique:3
-	media-libs/libsamplerate
-	sci-libs/fftw:3.0
-	sci-libs/gsl
-	sci-libs/itpp
-	x11-libs/gtk+:3
-	x11-libs/vte:="
+RDEPEND="dev-lang/lua:*
+	 dev-libs/libconfig
+	 dev-libs/libunique:3
+	 media-libs/libsamplerate
+	 sci-libs/fftw:3.0
+	 sci-libs/gsl
+	 sci-libs/itpp
+	 x11-libs/gtk+:3
+	 x11-libs/vte:2.90"
+
```

Comment:
Please use only tab for indention. 

https://devmanual.gentoo.org/ebuild-writing/file-format/ > "Indenting and Whitespace"